pallets/ajuna-awesome-avatars/src/types/avatar/tools/v2/constants.rs

```diff
@@ -0,0 +1,219 @@
+use super::*;
```

Comment:
To be decided, maintain as constants or have it in storage, where we have the flexibility, to change on the fly.